### DISCUSSION TITLE
What the heck just happened to application size?

### DISCUSSION THREAD

**Discussion by jaw1976** [TV-Rename/tvrename#830] [Q&A]
As a long time user I just got asked by the application if I wanted to update. I noticed that it was a major version version upgrade and got a bit curious. Except that it apparently now will support movies also, I can't really say anything big jumped out at me. That is until I downloaded the new installer.

TVRename-3.3.1.exe      8392 kB
TVRename-4.4.7.exe  134826 kB

WTF? What just happened? Did you include an entire OS in the installation? Old installation uses 20MB disk, new one looks to want 460MB. An not a single comment about it anywhere?

**Comment by SirSparkles** [comment]
v3 used an old component based on IE 8 to handle the right hand side of the app. This limited the utility of the information about movies and TV shows.

v4 wanted to have the image carousels, and trailers. To do this we needed to upgrade to a chrome based solution. It's had a few teething troubles, but IMHO is looking good now.

**Reply by jaw1976** [reply]
So the application 24-doubled in size on disk, most likely with added RAM and CPU footprint for some bling? I mean the right side of the "My Shows/etc" which I suppose that you refer to is almost never used, and not anything one needs to bounce around or show trailers in. I already know my shows. I don't need to see a trailer when I have the whole thing. And instead of an old but fairly stable component you switched to an ever updating component that will require constant patching in itself and most likely whatever you use it for?

Now, I try to avoid criticizing volunteer work, but I think you went completely the wrong direction here. I'll stay with the old version until I probably end up needing to switch to another tool.

So long, and thanks for all the fish

**Reply by SirSparkles** [reply]
Fair enough. The strategic move from IE9 an outdated component to Chrome was to remove the need for the work arounds and patching that were building up still supporting the old component. 

For more info see https://stackoverflow.com/questions/790542/replacing-net-webbrowser-control-with-a-better-browser-like-chrome
